ansible_collections/arista/avd/plugins/action/anta_workflow.py

```diff
@@ -56,6 +56,16 @@
     "ansible_httpapi_use_ssl",
 ]
 
+ANTA_CONNECTION_VARS = [
```

Comment:
anta_tags is not a connection var :)
```suggestion
ANTA_VARS = [
```